src/main/java/app/coronawarn/verification/service/TanService.java

```diff
@@ -53,7 +54,8 @@
   private static final Pattern TAN_PATTERN = Pattern.compile(TAN_TAN_PATTERN);
 
   // Tele-TANs are a shorter, easier to communicate form of TAN
-  private static final int TELE_TAN_LENGTH = 9;
+  @Value("${tan.tele.valid.length}")
+  private static int TELE_TAN_LENGTH = 9;
```

Comment:
statics should not (or can not!) be changed with @value
Just use VerificationApplicationConfig here also!

Reply:
No longer relevant